### DISCUSSION TITLE
Difficulties to install letsencrypt

### DISCUSSION THREAD

**Discussion by Siteac** [swizzin/swizzin#750] [Q&A + Help] [answered]
Hi everybody,

I have some troubles to install letsencrypt. 
I got this message in swizzin.log :

> [Sun 22 Aug 2021 10:42:18 PM CEST] Using CA: https://acme-v02.api.letsencrypt.org/directory
> [Sun 22 Aug 2021 10:42:18 PM CEST] Single domain='thisismydomain.com'
> [Sun 22 Aug 2021 10:42:18 PM CEST] Getting domain auth token for each domain
> [Sun 22 Aug 2021 10:42:21 PM CEST] Getting webroot for domain='thisismydomain.com'
> [Sun 22 Aug 2021 10:42:21 PM CEST] Verifying: thisismydomain.com
> [Sun 22 Aug 2021 10:42:21 PM CEST] Nginx mode for domain:thisismydomain.com
> [Sun 22 Aug 2021 10:42:21 PM CEST] Can not find conf file for domain thisismydomain.com
> [Sun 22 Aug 2021 10:42:21 PM CEST] Please add '--debug' or '--log' to check more details.
> [Sun 22 Aug 2021 10:42:21 PM CEST] See: https://github.com/acmesh-official/acme.sh/wiki/How-to-debug-acme.sh

Just installed nginx, deluge, sonarr, radarr and jackett without problems.

Do you have any idea ? 

Thank you very much for your help :)

**Comment by bakerboy448** [comment, marked as answer]
pinned GHI
https://github.com/swizzin/swizzin/issues/742

**Reply by Siteac** [reply]
Resolved my issue. Thank you.
